README.md

```diff
@@ -509,6 +509,7 @@ The `npm` utility must be installed on the system.
 
 | Name                | Description                                                         |
 | ------------------- | ------------------------------------------------------------------- |
+| `NPM_TOKEN`         | An automation token allowed to publish. (See https://bit.ly/3gSrVyT)|
```

Comment:
Let's have a direct link to the docs?

Reply:
This _is_ a link to the docs? Just shortened it as the long version is https://docs.npmjs.com/creating-and-viewing-access-tokens